server/src/main/java/io/littlehorse/server/streams/topology/core/MetadataCommandExecution.java

```diff
@@ -81,4 +87,17 @@ private AuthorizationContext authContextFor(TenantIdModel tenantId, PrincipalIdM
         // fine-grained authorization
         return new AuthorizationContextImpl(principalId, tenantId, List.of(), false);
     }
+
+    public Collection<PrincipalIdModel> adminPrincipalIds() {
+        Attribute tagAdminAttribute = new Attribute("isAdmin", Boolean.TRUE.toString());
+        Set<PrincipalIdModel> result = this.metadataManager()
+                .tagScan(PrincipalModel.getTypeEnum(PrincipalModel.class), List.of(tagAdminAttribute))
```

Comment:
`.tagScan` should be deprecated now. you can either modify the tagScan method in order to close the iterator or use other mechanism to execute the range scan

Reply:
I see, I copied this and pasted it from the other place. Will fix